presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowStatsRewrite.java

```diff
@@ -314,18 +327,47 @@ private TableHandle getTableHandle(ShowStats node, QualifiedName table)
             ImmutableList.Builder<String> columnNamesBuilder = ImmutableList.builder();
             columnNamesBuilder.add("column_name");
             columnNamesBuilder.addAll(statisticsNames);
+            columnNamesBuilder.add("low_value");
+            columnNamesBuilder.add("high_value");
             return columnNamesBuilder.build();
         }
 
-        private static Row createStatsRow(Optional<String> columnName, List<String> statisticsNames, Map<String, Estimate> columnStatisticsValues)
+        private Row createColumnStatsRow(String columnName, Type columnType, List<String> statisticsNames, ColumnStatistics columnStatistics)
         {
             ImmutableList.Builder<Expression> rowValues = ImmutableList.builder();
-            Expression columnNameExpression = columnName.map(name -> (Expression) new StringLiteral(name)).orElse(new Cast(new NullLiteral(), VARCHAR));
+            rowValues.add(new StringLiteral(columnName));
+            RangeColumnStatistics rangeStatistics = columnStatistics.getOnlyRangeColumnStatistics();
+            Map<String, Estimate> statisticsValues = rangeStatistics.getStatistics();
+            for (String statName : statisticsNames) {
+                rowValues.add(createStatisticValueOrNull(statisticsValues, statName));
+            }
+            rowValues.add(asVarcharLiteral(columnType, rangeStatistics.getLowValue()));
+            rowValues.add(asVarcharLiteral(columnType, rangeStatistics.getHighValue()));
+            return new Row(rowValues.build());
+        }
+
+        private Expression asVarcharLiteral(Type valueType, Optional<Object> value)
+        {
+            if (!value.isPresent()) {
+                return new Cast(new NullLiteral(), VARCHAR);
+            }
+            FunctionRegistry functionRegistry = metadata.getFunctionRegistry();
```

Comment:
It's fine. I wonder though if it not something that we would like to extract or we don't have that extracted already. It seems that something that would be universally useful.